client/src/components/Forms/signup/index.jsx

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import axios from 'axios';
+import {
+  Modal,
+  Button,
+  Form,
+  Input,
+  message,
+} from 'antd';
+import logo from '../../../assets/images/logo.png';
+import './Signup.css';
+
+function Signup() {
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [fullName, setFullName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirm, setConfirm] = useState('');
+  const [rool] = useState('');
+
+  const signUp = async () => {
+    try {
+      await axios.post('/api/signup', {
+        fullName,
+        email,
+        password,
+        rool,
+      });
+      setIsModalVisible(false);
+      message.success('Welcome to Nova');
+    } catch (err) {
+      message.error('Sorry, try again');
+      console.log(err);
```

Comment:
Remove the console, and we need to send the error that we send from the backend so the user nhow exactly whats wrong